options.js

```diff
@@ -1,21 +1,45 @@
 let page = document.getElementById('colorSelector');
 
-const googleGreen = "#009688";
-const googleAquaBlue = "#00BBD3";
-
 const input = document.getElementById("colorInput");
-let submitButton = document.getElementById("submitButton");
+getExtensionColor().then(color => setExtensionColor(color));
```

Comment:
You shouldn't set the extension color like this, you're just loading the configuration and setting it again.

Reply:
This was done to set the extension color when there is no user data, but you are right, there is a better way to solve this.